lib/console/email.ex

```diff
@@ -86,6 +86,19 @@ defmodule Console.Email do
     |> render(:invitation_email)
   end
 
+  def resend_invitation_email(%Invitation{email: email, token: token, role: role}, inviter_email, %Organization{name: organization_name}) do
```

Comment:
same function as above but since the invitee is the one logged in, I don't have `conn.current_user` to pass as `User%{}` so using the email directly here.

Reply:
You can just create a fake %User{email: inviter_email} object where this is being called and you can use the above function